src/ansys/fluent/core/utils/__init__.py

```diff
@@ -6,6 +6,8 @@
 
 logger = logging.getLogger("pyfluent.general")
 
+from .search import _search  # noqa: F401
```

Comment:
How is this used here? This module would be independent of any pyfluent code

Reply:
We are getting circular import error so to resolve this we have used it here.

Reply:
We have 2 ways to use `_search` - 
1. import _search in `ansys/fluent/core/__init__.py` (last time we have decided to remove it from here) or 
2. import _search `ansys/fluent/core/utils/__init__.py` - This resolves circular import error

Is there any other way to use `_search` in `flobject.py` ? because `flobject` is imported in `search.py` hence we can not import it directly.

Reply:
I thought `_search` is a part of `utils` therefore we can import it on `utils` level because we are doing the same for launcher. We are importing all required launcher code in `ansys/fluent/core/launcher/__init__.py` from all submodules under ``ansys/fluent/core/launcher` directory.

Reply:
Thanks for clarifying, yeah, it is fine. At some point we shall remove the flobject depedencies on search.py (it is used only in the old function).

Reply:
Sure.